src/unfold.js

```diff
@@ -3,35 +3,37 @@ var _curry2 = require('./internal/_curry2');
 
 /**
  * Builds a list from a seed value. Accepts an iterator function, which returns
- * either false to stop iteration or an array of length 2 containing the value
- * to add to the resulting list and the seed to be used in the next call to the
- * iterator function.
+ * either an empty array to stop iteration, or a singleton array containing an
+ * array of length 2, with the value that's added to the resulting list and the
+ * seed to be used in the next call of the iterator function.
  *
  * The iterator function receives one argument: *(seed)*.
  *
  * @func
  * @memberOf R
  * @since v0.10.0
  * @category List
- * @sig (a -> [b]) -> * -> [b]
+ * @sig (a -> Maybe [b]) -> * -> [b]
```

Comment:
It has been taking me a while to understand that you were referring to the usage order of `a` and `b` within my previous proposals 😅 now I see it, and it makes sense! (since what's being pass to the function is the second value of the previous iteration).